css/style.css

```diff
@@ -0,0 +1,397 @@
+    max-width: 50%;
+}
+
+.col-7 {
+    flex-basis: 58.333333%;
+    max-width: 58.333333%;
+}
+
+.col-8 {
+    flex-basis: 66.666667%;
+    max-width: 66.666667%;
+}
+
+.col-9 {
+    flex-basis: 75%;
+    max-width: 75%;
+}
+
+.col-10 {
+    flex-basis: 83.333333%;
+    max-width: 83.333333%;
+}
+
+.col-11 {
+    flex-basis: 91.666667%;
+    max-width: 91.666667%;
+}
+
+.col-12 {
+    flex-basis: 100%;
+    max-width: 100%;
+}
+
+.w-25 {
+    width: 25%;
+}
+
+.w-50 {
+    width: 50%;
+}
+
+.w-75 {
+    width: 75%;
+}
+
+.w-100 {
+    width: 100%;
+}
+
+.flex {
+    display: flex;
+}
+
+.justify-content-end {
+    justify-content: flex-end;
+}
+
+.align-items-end {
+    align-items: flex-end;
+}
+
+.justify-content-between {
+    justify-content: space-between;
+}
+
+.p-1 {
+    padding: 1rem;
+}
+
+.p-2 {
+    padding: 2rem;
+}
+
+.pl-1 {
+    padding-left: 1rem;
+}
+
+.pl-2 {
+    padding-left: 2rem;
+}
+
+.pr-1 {
+    padding-right: 1rem;
+}
+
+.pr-2 {
+    padding-right: 2rem;
+}
+
+.pt-1 {
+    padding-top: 1rem;
+}
+
+.pt-2 {
+    padding-top: 2rem;
+}
+
+.pb-1 {
+    padding-bottom: 1rem;
+}
+
+.pb-2 {
+    padding-bottom: 2rem;
+}
+
+.m-1 {
+    margin: 1rem;
+}
+
+.m-2 {
+    margin: 2rem;
+}
+
+.ml-1 {
+    margin-left: 1rem;
+}
+
+.ml-2 {
+    margin-left: 2rem;
+}
+
+.mr-1 {
+    margin-right: 1rem;
+}
+
+.mr-2 {
+    margin-right: 2rem;
+}
+
+.mt-1 {
+    margin-top: 1rem;
+}
+
+.mt-2 {
+    margin-top: 2rem;
+}
+
+.mb-1 {
+    margin-bottom: 1rem;
+}
+
+.mb-2 {
+    margin-bottom: 2rem;
+}
+
+/* end of layout */
+
+/* html */
+
+html {
+    font-family: sans-serif;
+    -ms-text-size-adjust: 100%; 
+    -webkit-text-size-adjust: 100%; 
+}
+html {
+    display: block;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+/* end of html */
+
+/* head */
+
+head {
+    display:none;
+}
+
+/* end of head */
+
+/* body */
+
+body {
+    margin: 0;
+    display: block;
+}
+
+/* end of body */
+
+/* header */
+
+.header-image {
+    padding: 2rem;
+    margin-bottom: 1rem;
+    background: #3a2929;
+    display: flex;
+    flex-wrap: wrap;
+} 
+
+.w-100 {
+    width: 5%;
+}
+
+.numb1 {
+    display: flex;
+    justify-content: flex-end;
+    flex-basis: 88.666667%;
+    max-width: 88.666667%;
+}
+    
+.number {
+    padding: 1rem;
+    background-color: #ad8383;
+}
+/* end of header */
+
+/* nav bar */
+
+.navbar {
+    padding-left: 1rem;
+    padding-right: 2rem;
+    padding-top:   1rem;
+    padding-bottom: 2rem;
+    margin-bottom: 1rem;
+    background: #ad8383;
+}
+
+.nav-list {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+ul {
+    list-style-type: none;
+    padding-left: 0;
+}
+
+ul li a {
+    text-decoration: none;
+}
+
+/* end of nav bar */
+
+/* main:section 1 */
+
+#info .row {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.info {
+    padding: 2rem;
+    flex-basis: 33.333333%;
+    max-width: 33.333333%;
+    background: #d1cecd;
+}
+
+.text {
+    margin-bottom: 2rem;
+
+}
+
+.sec-1a {
```

Comment:
The .col-8 class earlier in the file contains these same style rules so I think it would been best to just use that and remove this statement